tests/certification/bindings/azure/eventhubs/send-iot-device-events.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+# ------------------------------------------------------------
+# Copyright 2021 The Dapr Authors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------
+
+set -e
+
+if [[ -z "${IOT_HUB_NAME}" ]]; then
+    echo "ERROR: IOT_HUB_NAME environment variable not defined."
+    exit 1
+fi
+if [[ -z "${AZURE_CREDENTIALS}" ]]; then
```

Comment:
Please remove all uses of AZURE_CREDENTIALS in this script and instead use the approach as seen here:

https://github.com/dapr/components-contrib/blob/master/tests/certification/pubsub/azure/eventhubs/send-iot-device-events.sh#L33